src/css/custom.css

```diff
@@ -75,6 +89,10 @@ html[data-theme='dark'] {
   line-height: 1em;
 }
 
+.theme-doc-markdown {
+  --ifm-link-decoration: underline;
```

Comment:
Is this required for a perfect score?

Reply:
Hey yes, the links need to be underlined to meet 100% accessibility requirements

Reply:
Got it, thanks for the elaboration